test/test_cli_integration.py

```diff
@@ -108,18 +109,24 @@ def delete_policy_from_string(self, policy):
 
         return output
 
-    def get_variable(self, *variable_ids):
+    def get_variable(self, variable_ids):
```

Comment:
why has the `*` been removed?

Reply:
So it would get an array.

Reply:
You can still pass in an array.

```
def myFun(*argv): 
	for arg in argv: 
		print(arg) 

myFun(['Hello', 'Welcome', 'to', 'GeeksforGeeks']) 
```
will print each arg but notice how I kept the *
The [*arg ](https://www.geeksforgeeks.org/args-kwargs-python/#:~:text=The%20special%20syntax%20*args%20in,used%20with%20the%20word%20args.)just means you can pass any number of arguments without having to define them all as parameters explicitly